packages/reapit-connect-management/src/components/ui/apps/app-installation-manager.tsx

```diff
@@ -29,7 +28,7 @@ const AppInstallationManager: React.FC<AppInstallationManagerProps> = ({ app }:
 
   const { data: installations, isValidating: installationsValidating, revalidate: revalidateInstallations } = useSWR<
     InstallationModelPagedResult
-  >(`${URLS.INSTALLATIONS}/?AppId=${app.id}&IsInstalled=true`)
+  >(`${URLS.INSTALLATIONS}/?AppId=${app.id}&IsInstalled=true&pageSize=999`)
```

Comment:
I added this line as this request was paginated at 15 so would only have returned that many installs